### PR TITLE
Feat: Added routing mechanism

### PR DESCRIPTION
**Pull request by lightme-fan**
Hello team,

I have finished the routing mechanism. Please have a look at it.

Thanks in advance,

- Branch: [https://github.com/onja-org/vamalaka-fannar/tree/routing-mechanism](https://github.com/onja-org/vamalaka-fannar/tree/routing-mechanism)
- Clickup: [https://app.clickup.com/t/45e4af](https://app.clickup.com/t/45e4af)
- Refs: #45e4af

### CODE REVIEW

**Review by mitia-anah — APPROVED** [review]

**Review by Tiaraha-funny — APPROVED** [review]

File: src/components/HeaderNavLink/HeaderNavLink.tsx [hunk 1]
```diff
@@ -22,23 +23,23 @@ export const loggedIn = [
...
   {
-    path: '#login',
+    path: 'login',
     imgSrc: signin,
     alt: 'Log in',
     text: 'Log in',
   },
   {
-    path: '#account',
+    path: 'registering-account',
```
Comment (andrzej-onja) [inline src/components/HeaderNavLink/HeaderNavLink.tsx:35]: just "register" please

File: src/components/HeaderNavLink/HeaderNavLink.tsx [hunk 2]
```diff
@@ -75,8 +78,9 @@ const Item = styled.li`
       }
     }
   }
+
   &:first-of-type {
-    @media (max-width: 900px) {
+    @media (max-width: 1000px) {
```
Comment (andrzej-onja) [inline src/components/HeaderNavLink/HeaderNavLink.tsx:83]: https://codesandbox.io/s/media-queries-in-styled-components-typescript-forked-o4tt0

Comment (andrzej-onja) [reply src/components/HeaderNavLink/HeaderNavLink.tsx:83]: please implement as in the message https://onja20.slack.com/archives/C01BXQRM6LV/p1627044969006300

File: src/containers/RoleSelectionContainer.tsx [hunk 3]
```diff
@@ -24,20 +25,24 @@ export const RoleSelectionContainer = () => {
...
+            </Link>
           </div>
           <div>
-            <Option
-              label='Seller'
-              text='Own or belong to a company, this is for you.'
-              src={briefcaseIcon}
-              alt='Briefcase icon'
-            />
+            <Link to='/registering-account'>
```
Comment (andrzej-onja) [inline src/containers/RoleSelectionContainer.tsx:38]: We should create a file with enum RoutesInfo { 
LOGIN='/login'
...
}

File: src/pages/RoleSelection.tsx [hunk 4]
```diff
@@ -1,11 +1,28 @@
...
+        <Route path='/' exact>
+          <RoleSelectionContainer />
+        </Route>
+        <Route path='/language'>
+          <LanguagePage />
+        </Route>
+        <Route path='/login'>
+          <LoginPage />
+        </Route>
+        <Route path='/registering-account'>
```
Comment (andrzej-onja) [inline src/pages/RoleSelection.tsx:22]: use enum to get route paths

File: src/paths/index.tsx [hunk 5]
```diff
@@ -0,0 +1,10 @@
+export enum Paths {
+  DEFAULT = '/',
+  LOGIN = '/login',
+  REGISTER_ACCOUNT = '/register-account',
+  LANGUAGE = '/language',
+}
+
+export interface PathProps {
+  Path: Paths
+}
```
Comment (andrzej-onja) [inline src/paths/index.tsx:10]: PathProps is not used, remove

**Review by andrzej-onja — APPROVED** [review]

**Pull request merged**